File: app/src/main/java/ru/levprav/shaint/ui/listproducts/view/ListProductsView.java
package ru.levprav.shaint.ui.listproducts.view;

import java.util.List;

import ru.levprav.shaint.base.BaseView;
import ru.levprav.shaint.domain.model.Product;

public interface ListProductsView extends BaseView {
    void showProducts(List<Product> products);
}

File: app/src/main/java/ru/levprav/shaint/ui/listproducts/view/ListProductsView.kt
package ru.levprav.shaint.ui.listproducts.view

import ru.levprav.shaint.base.BaseView
import ru.levprav.shaint.ui.listproducts.view.adapter.RecyclerViewItem

interface ListProductsView : BaseView {
    fun showPopularProducts(products: List<RecyclerViewItem>)
    fun showYourProducts(products: List<RecyclerViewItem>)
}